debian/mariadb-server.mariadb.init

```diff
@@ -176,8 +176,21 @@ case "${1:-''}" in
     # Start MariaDB!
     /usr/bin/mariadbd-safe "${@:2}" 2>&1 >/dev/null | $ERR_LOGGER &
 
-    for _ in $(seq 1 "${MYSQLD_STARTUP_TIMEOUT:-30}")
```

Comment:
Why not use [seq](https://manpages.debian.org/unstable/coreutils/seq.1.en.html), why need to stop using it?

Reply:
Why not use [seq](https://manpages.debian.org/unstable/coreutils/seq.1.en.html), why need to stop using it?